improver/cli/combine.py

```diff
@@ -42,6 +42,7 @@ def process(
     new_name=None,
     use_midpoint=False,
     check_metadata=False,
+    broadcast_to_coord=None,
```

Comment:
Here there's just one item, but you've edited `CubeCombiner` to accept a list.  Any particular reason?

Reply:
Working out how to make this a list made my head hurt and the use case doesn't require it. I could restrict the plugin to just take one I suppose. Seems a bit of a backward step though.

Reply:
Yes perhaps your right.  I was aiming for consistency between plugin and CLI, but actually having the extra functionality in the plugin doesn't add a lot of complexity.  So this is probably fine.

Agree to keep the interface here simple, if one coord is what we expect, then that's what we should pass in.